akasha-vladan-main/README.md

```diff
@@ -0,0 +1,3 @@
+# Microverse Trial #1
+
+Project "Build a Tribute Page" from freeCodeCamp for Microverse trial
```

Comment:
I think it would be better if you could add a screenshot of your project so as to make it look more professional.